runtime/v2/binary.go

```diff
@@ -74,7 +74,7 @@ func (b *binary) Start(ctx context.Context, opts *types.Any, onClose func()) (_
 	if err != nil {
 		return nil, err
 	}
-	f, err := openShimLog(ctx, b.bundle, client.AnonDialer)
+	f, err := openShimLog(context.Background(), b.bundle, client.AnonDialer)
```

Comment:
@darfux: This change broke support on Windows, as `openShimLog` in shim_windows.go requires a Namespace.

Reply:
Sorry for missing the windows condition😂. I'm trying to fix it on https://github.com/containerd/containerd/pull/3929, PTAL